RecoCTPPS/Configuration/python/RecoCTPPS_EventContent_cff.py

```diff
@@ -17,7 +17,8 @@
     # timing diamonds
     'keep TotemFEDInfos_ctppsDiamondRawToDigi_*_*',
     'keep CTPPSDiamondDigiedmDetSetVector_ctppsDiamondRawToDigi_*_*',
-    'keep TotemVFATStatusedmDetSetVector_ctppsDiamondRawToDigi_*_*'
+    'keep TotemVFATStatusedmDetSetVector_ctppsDiamondRawToDigi_*_*',
+    'keep CTPPSDiamondRecHitedmDetSetVector_ctppsDiamondRecHit_*_*',
```

Comment:
the name has changed to ctppsDiamondRecHits
please update the product selection